app/uk/gov/hmrc/emcstfe/models/request/SubmitCancellationOfMovementRequest.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfe.models.request
+
+import uk.gov.hmrc.emcstfe.models.auth.UserRequest
+import uk.gov.hmrc.emcstfe.models.cancellationOfMovement.SubmitCancellationOfMovementModel
+import uk.gov.hmrc.emcstfe.models.common.DestinationType.{ExemptedOrganisations, Export}
+
+import scala.xml.Elem
+
+case class SubmitCancellationOfMovementRequest(body: SubmitCancellationOfMovementModel)
```

Comment:
I think you're missing the unit spec for this model

Reply:
Is that failing test meant to be testing the `messageRecipent` instead of the `messageSender`? The match case only gets used on the recipient side currently